pythermalcomfort/models/bfu_rest.py

```diff
@@ -0,0 +1,238 @@
+from __future__ import annotations
+
+import numpy as np
+
+from pythermalcomfort.classes_input import BFU_rest_Groups, BFU_rest_Inputs
+from pythermalcomfort.classes_return import BFU_rest as BFURestResult
+
+
+def morris_lph_fans(
+    tdb: float | list[float] | np.ndarray,
+    rh: float | list[float] | np.ndarray,
+    group: str = BFU_rest_Groups.young.value,
+    round_output: bool = True,
+) -> BFURestResult:
+    """Calculate the net heat storage between fans vs still air using a biophysical
+    model for resting individuals.
+
+    Parameters
+    ----------
+    tdb : float or list of floats
+        Dry bulb air temperature, :math:`°C`.
+    rh : float or list of floats
+        Relative humidity, :math:`%`.
+    group : str, optional
+        Population group considered in the model. Choices are ``'YNG'``, ``'OLD'``, and ``'MEDS'``. Defaults to ``'YNG'``.
+    round_output : bool, optional
+        If ``True`` the returned values are rounded to one decimal place. Defaults to ``True``.
+
+    Returns
+    -------
+    BFU_rest
+        A dataclass containing the net difference in heat storage and evaporative requirements between fans and still air. See :py:class:`~pythermalcomfort.classes_return.MorrisLPHFans` for details.
+    """
+
+    def _psat_kpa_air(t_c: np.ndarray) -> np.ndarray:
+        """Saturation vapour pressure over water, [kPa]."""
+        return np.exp(18.956 - (4030.18 / (t_c + 235))) / 10
+
+    def _icl_from_rcl(rcl_m2k_w: float) -> float:
```

Comment:
why all these functions are defined within a function? A bit hard to follow the code